config/onboarding/hertfordshire-production.yaml

```diff
@@ -30,9 +35,105 @@ schools:
     - 142902
     - 145051
     - 145271
+  essex:
+    - 115235
```

Comment:
I've just tried the first URN and it turns out it's a closed establishment on GIAS 🤦 
It does have a linked school – I'll see if I can clean this list up over the weekend.

Reply:
Addressed in [0733ad8](https://github.com/nhsuk/manage-vaccinations-in-schools/pull/3388/commits/0733ad879fcc8c285a6f9f4394ef04f17066e4c9)